interactive/run.py

```diff
@@ -0,0 +1,140 @@
+import os
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+from services import opencodelists
+
+
+PROJECT_YAML = """
+version: '3.0'
+
+expectations:
```

Comment:
I haven't tried it, but it's possible we don't need the expectations/population_size lines in the project.yaml

Reply:
Hah. 

So, technically true atm I think, but once @ghickman's pipeline validation changes land, I think we'll need it.